CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+##  v1.10.0 (12/14/2020)
+
+**Common**
+
+* Added a shared_mutex class.
+* Added status logging of the network requests on iOS platform.
+* Fixed the proxy warming message on Android platform.
+* Added `Open` and `Close` API to `olp::cache::DefaultCache`. It can be used to open or close individual caches.
+* `olp::client::ApiLookupClient` now caches the `olp::client::OlpClient` instances internaly.
```

Comment:
```suggestion
* `olp::client::ApiLookupClient` now caches the `olp::client::OlpClient` instances internaly. This will enable us to merge same URL requests inside the `olp::client::OlpClient` instance.
```